html/cross-origin-embedder-policy/header-parsing.https.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+  <meta charset="utf-8">
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+<body>
+<script>
+'use strict';
+function createIframe(t, values) {
+  const parent = document.createElement('iframe');
+  const child = document.createElement('iframe');
+  const params = new URLSearchParams(values.map((value) => ['value', value]));
+  parent.setAttribute('src', `resources/empty-coep.py?${params}`);
+  document.body.appendChild(parent);
+  t.add_cleanup(() => parent.remove());
+
+  return new Promise((resolve, reject) => {
+      parent.onload = resolve;
+      parent.onerror = reject;
+    })
+    .then(() => {
+        child.setAttribute('src', '/common/blank.html');
+        parent.contentDocument.body.appendChild(child);
+        return new Promise((resolve) => child.onload = resolve);
+      })
+    .then(() => child);
+}
+
+[
+  [],
+  [''],
+  ['jibberish'],
+  ['require\u2014corp'], // non-ASCII character (em dash)
+  ['require-corp;'],
+  ['require-corp\0'],
+  ['Require-corp'],
+  ['require-corp; foo=bar'],
+  ['"require-corp"'], // HTTP structured header "string" item
+  ['*cmVxdWlyZS1jb3Jw*'], // HTTP structured header "byte sequence" item
+  ['require-corp require-corp'],
+  ['require-corp,require-corp'],
+  ['require-corp;require-corp'],
```

Comment:
This may also be valid as a parameter; I'm not sure

Reply:
https://tools.ietf.org/html/draft-ietf-httpbis-header-structure-14#section-3.1.2
https://tools.ietf.org/html/draft-ietf-httpbis-header-structure-14#section-4.2.3.2
seems to say that the value is indeed optional.